src/main/java/net/sf/jabref/logic/bibtexkeypattern/BibtexKeyPatternUtil.java

```diff
@@ -519,6 +521,13 @@ public static String applyModifiers(String label, List<String> parts, int offset
                     }
                     resultingLabel = abbreviateSB.toString();
 
+                } else if ("camel".equals(modifier)) {
```

Comment:
Would it make sense to support all Formatters as modifiers? If this is the case, I would add a method `Optional<Formatter> gefFormatterByTitle(String title)` to `Formatters` and use this here.
I think this also results in a more cleaner design.